packages/wasm-dpp/test/unit/document/DocumentFactory.spec.js

```diff
@@ -287,9 +287,15 @@ describe('DocumentFactory', () => {
       const [newDocument] = await getDocumentsFixture(dataContract);
       newDocument.setData({ lastName: 'Keck' });
 
+      const identityId = newDocument.getOwnerId();
+
       const stateTransition = factory.createStateTransition({
         create: documents,
         replace: [newDocument],
+      }, {
+        [identityId.toString()]: {
```

Comment:
We can have only one identity for ST, so we just need to pass only contract to nonce

Reply:
We agreed that this can be refactored in future